solutions/kubernetes/ceph-deploy-functions.sh

```diff
@@ -39,6 +40,8 @@ where,
     --deploy-mds - Deploy Ceph Metadata Service daemon on primary node.
     --deploy-fs - Deploy Ceph FS daemon on primary node.
     --deploy-rgw - Deploy Ceph Rados Gateway daemon on primary node.
+    --install-prereq-image - Setup prerequisites for Ceph docker deployment.
+    --deploy-ceph-image- Deploy Ceph in docker.
```

Comment:
should mention `deploy in docker` or `deploy in container`?
same for above line as well

Reply:
Done